scripts/tests/test_contabo_host_key_workflows.py

```
@@ -0,0 +1,154 @@
+      write_executable(
+          fake_bin / "ssh-keyscan",
+          f"""\
+          #!/usr/bin/env bash
+          touch {sentinel}
+          exit 0
+          """,
+      )
+      env = os.environ.copy()
+      env["PATH"] = f"{fake_bin}{os.pathsep}{env['PATH']}"
+
+      result = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)